src/torrent/connection_manager.h

```diff
@@ -165,7 +167,7 @@ class LIBTORRENT_EXPORT ConnectionManager {
   slot_throttle_type& address_throttle()  { return m_slot_address_throttle; }
 
   // For internal usage.
-  Listen*             listen()            { return m_listen; }
+  const std::unique_ptr<Listen>& listen() { return m_listen; }
```

Comment:
This is incorrect, we don't pass the unique_ptr to the outside, it should just return a pointer.

Reply:
Removed.